app/src/main/java/net/opendasharchive/openarchive/util/Prefs.kt

```diff
@@ -124,4 +125,14 @@ object Prefs{
 
             prefs?.edit()?.putString(PROOFMODE_ENCRYPTED_PASSPHRASE, passphrase)?.apply()
         }
+
+    var theme: String
+        get() = prefs?.getString(PREF_THEME, null) ?: "system"
+        set(value) {
+            var v: String = value
```

Comment:
Useless explicit type.

Reply:
When I remove the type declaration I get an compiler error :worried: 

```
This property must either have a type annotation, be initialized or be delegated
```

I'm not really sure but I think initializing probably would be an error or this context and I don't know what delegated means.